### PR TITLE
CI: change to use ubuntu-latest GitHub hosted runner

### PR DESCRIPTION
**Pull request by rmstoi**

### CODE REVIEW

**Comment by poussa** [comment]
@rmstoi this is not building anything yet, correct? Do yo plan to update the PR for the build or is that separate ?

**Comment by rmstoi** [comment]
@poussa I disabled build, to test installation of kubebuilder (used by make test) on runner, I will update PR later. Hence, please DNM yet.

**Comment by rmstoi** [comment]
All issues fixed, PR can be merged

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -28,6 +28,14 @@ jobs:
             dep ensure
         fi
 
+    - name: Get kubebuilder
+      run: |
+        os=$(go env GOOS)
+        arch=$(go env GOARCH)
+        curl -L https://go.kubebuilder.io/dl/2.3.1/${os}/${arch} | tar -xz -C /tmp/
```
Comment (ipuustin) [inline .github/workflows/ci.yml]: Please check the sha256 of the file you download.

**Review by ipuustin — CHANGES_REQUESTED** [review]
One small nit, otherwise looks good to me.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (rmstoi) [reply .github/workflows/ci.yml]: Done

**Pull request merged**